PowerEditor/src/WinControls/SplitterContainer/Splitter.cpp

```diff
@@ -389,9 +379,10 @@ LRESULT CALLBACK Splitter::spliterWndProc(UINT uMsg, WPARAM wParam, LPARAM lPara
 
 		case WM_LBUTTONUP:
 		{
-			if (!_isFixed)
+			if (!_isFixed && _isLeftButtonDown)
 			{
 				ReleaseCapture();
+				_isLeftButtonDown = false;

```

Comment:
I may be wrong in observation here. `_isLeftButtonDown` may remain true on mouse button up if `_isFixed` is true

Reply:
Hmm, no. `_isLeftButtonDown` is set to `true` only if `_isFixed` is not true. [Look here](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/5554/files#diff-a39360c3bc45e15d195381c6bfdcc4a2R274)